packages/siop-oid4vp/lib/types/VpJwtVerifier.ts

```diff
@@ -96,10 +96,10 @@ export const getRequestObjectJwtVerifier = async (
     if (jwt.payload.redirect_uri && jwt.payload.redirect_uri !== clientId) {
```

Comment:
There probably should be a else if here, that does the exact same check but then for jwt.payload,response_uri (the new name) instead of redirect_uri

Reply:
Is client_id_scheme still redirect_uri in case  jwt.payload,response_uri is used?
I don't see redirect_uri as valid value for client_id_scheme

Reply:
> There probably should be a else if here, that does the exact same check but then for jwt.payload,response_uri (the new name) instead of redirect_uri

This was redolved BTW